python-clusters/attach-eks-cluster/cluster.py

```diff
@@ -68,6 +68,9 @@ def start(self):
         with open(kube_config_path, 'w') as f:
             f.write(kube_config_str)
 
+        logging.info("Cluster attached. Cluster configuration:")
+        logging.info(kube_config_str)
```

Comment:
Feel free to ignore as it might be overkill, but you could also log the original `cluster_info` response.

Reply:
done